CHANGELOG.md

```diff
@@ -269,7 +350,7 @@ Same changes as in `@mui/x-date-pickers@8.6.0`.
 - [charts] Improve touch behavior for polar axis (#18531) @JCQuintas
 - [charts] Add `isElementInside` helper (#18530) @JCQuintas
 
-#### `@mui/x-charts-pro@8.6.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link "Pro plan")
+#### `@mui/x-charts-pro@8.6.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
```

Comment:
I think you might have run prettier or so on md files

Reply:
Yes, I did. Is that problematic?

Reply:
note really, but we typically leave the previous CHANGELOG entries untouched